doc/README.BBG.FDW.md

```diff
@@ -0,0 +1,52 @@
+# Setup foreign data wrapper data library import for cartodb-org repo
```

Comment:
Some more detailed notes on devops config steps to get this working on staging/production servers. Is there a better place to put these?